src/index.js

```diff
@@ -19,6 +20,45 @@ const client = new Client();
       console.log('>> ', message.content);
     });
 
+    // The code below listens for reactions to any message in the server and if
+    // a reaction is equal to the specified trigger reaction (in this case '🤖'),
+    // then is attempts to format the message. To get a detailed explanation of the
+    // code below visit: https://discordjs.guide/popular-topics/reactions.html#listening-for-reactions-on-old-messages
+
+    client.on('messageReactionAdd', async (reaction) => {
+      if (reaction.partial) {
+        try {
+          await reaction.fetch();
+        } catch (error) {
+          try {
+            reaction.message.channel.send(
+              'Something went wrong! Failed to format code :('
+            );
+          } catch (error) {
+            // maybe write an error log or something
+          }
+
+          return;
+        }
+      }
+
+      if (reaction.emoji.name === '🤖') {
+        try {
+          await reaction.message.reactions.removeAll();
+
+          addFormatting.command(reaction.message);
+        } catch (error) {
+          try {
+            reaction.message.channel.send(
+              'Something went wrong! Failed to format code :('
+            );
+          } catch (error) {
+            // maybe write an error log or something
+          }
```

Comment:
Same as the other comment on nested try/catch blocks

Reply:
Done...